Shadowgraphy plugin is not excluded when openPMD is not available

When compiling PIConGPU with fftw3 avaialbe but not openPMD-api, the compiler will fail with the following error:
```
In file included from /p/project/___________/_______/picongpu/include/picongpu/../picongpu/plugins/shadowgraphy/Shadowgraphy.hpp:26,
                 from /p/project/___________/_______/picongpu/include/picongpu/../picongpu/plugins/PluginController.hpp:47,
                 from /p/project/___________/_______/picongpu/include/picongpu/../picongpu/unitless/starter.unitless:23,
                 from /p/project/___________/_______/picongpu/include/picongpu/../picongpu/simulation_defines.hpp:58,
                 from /p/project/___________/_______/picongpu/include/picongpu/main.cpp:23:
/p/project/___________/_______/picongpu/include/picongpu/../picongpu/plugins/common/openPMDAttributes.hpp:27:10: fatal error: openPMD/openPMD.hpp: No such file or directory
   27 | #include <openPMD/openPMD.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

It appears to be caused by a missing include guard.

--- a/include/picongpu/plugins/PluginController.hpp
+++ b/include/picongpu/plugins/PluginController.hpp
@@ -43,7 +43,7 @@
  */
 #include "picongpu/plugins/PngPlugin.hpp"
 
-#if(SIMDIM == DIM3 && PIC_ENABLE_FFTW3 == 1)
+#if(SIMDIM == DIM3 && PIC_ENABLE_FFTW3 == 1 && ENABLE_OPENPMD == 1)
 #    include "picongpu/plugins/shadowgraphy/Shadowgraphy.hpp"
 #endif
 
@@ -141,7 +141,7 @@ namespace picongpu
             EnergyFields,
             ChargeConservation,
             SumCurrents
-#if(SIMDIM == DIM3 && PIC_ENABLE_FFTW3 == 1)
+#if(SIMDIM == DIM3 && PIC_ENABLE_FFTW3 == 1 && ENABLE_OPENPMD == 1)
             ,
             plugins::multi::Master<plugins::shadowgraphy::Shadowgraphy>
 #endif

